SPECS/iptables/iptables.spec

```diff
@@ -96,9 +95,10 @@ rm -rf %{buildroot}/*
 %{_mandir}/man3/*
 
 %changelog
-* Sat May 09 00:21:34 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.8.3-3
-- Added %%license line automatically
-
+*   Thu Mar 11 2021 Rachel Menge <rachelmenge@microsoft.com> - 1.8.3-4
```

Comment:
Since you're here, may I ask you to also check the source's license and then add the "License verified" line to your changelog entry? If the license is incorrect, you'd also need to update the `License` tag, of course. Judging by the `%license` macro, you should find the info you need inside the `COPYING` file at the root level of the untared ("untarred"?) source. If there are more license-related files there, please also expand the `%license` macro.

Reply:
done :)